middleware/locals.js

```diff
@@ -4,120 +4,120 @@ const moment = require('moment');
 const { map, omitBy } = require('lodash');
 
 const { getCurrentUrl, getAbsoluteUrl, localify } = require('../modules/urls');
+const { REBRAND_SECRET } = require('../modules/secrets');
+const appData = require('../modules/appData');
 const routes = require('../controllers/routes');
 
-/**
- * Get normalised page title for metadata
- */
-function getMetaTitle(base, pageTitle) {
-    return pageTitle ? `${pageTitle} | ${base}` : base;
-}
+const cookies = config.get('cookies');
+const features = config.get('features');
 
 /**
  * Set request locals
  * - Local properties that depend on the request
  * - Local methods for use in views that depend on the request
  */
-module.exports = {
-    // Export for tests
-    getMetaTitle,
```

Comment:
Removing the test for this and inlining the function as we test the output of this more thoroughly as part of the end-to-end tests.